src/ansys/additive/core/parametric_study/parametric_runner.py

```diff
@@ -92,6 +97,18 @@ def simulate(
         view = df[
             (df[ColumnNames.STATUS] == SimulationStatus.PENDING) & df[ColumnNames.TYPE].isin(type)
         ]
+
+        if isinstance(simulation_ids, list) and len(simulation_ids) > 0:
+            simulation_ids_list = list()
+            for sim_id in simulation_ids:
+                if sim_id not in view[ColumnNames.ID].values:
+                    LOG.warning(f"Simulation ID '{sim_id}' not found in the parametric study")
+                elif sim_id in simulation_ids_list:
+                    LOG.warning(f"Simulation ID '{sim_id}' has already been added")
+                else:
+                    simulation_ids_list.append(sim_id)
+            view = df[df[ColumnNames.ID].isin(simulation_ids_list)]
```

Comment:
Be careful here. If you use `df`, you lose the filter already applied on line 98.
```suggestion
            view = view[view[ColumnNames.ID].isin(simulation_ids_list)]
```